addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-custom-action.js

```diff
@@ -0,0 +1,416 @@
+import Component from '@glimmer/component';
+import rdflib from 'browser-rdflib';
+import { v4 as uuidv4 } from 'uuid';
+import { tracked } from '@glimmer/tracking';
+import { A } from '@ember/array';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { scheduleOnce } from '@ember/runloop';
+
+import {
+  RDF,
+  XSD,
+  removeSimpleFormValue,
+} from '@lblod/submission-form-helpers';
+
+const MU = new rdflib.Namespace('http://mu.semte.ch/vocabularies/core/');
+const QB = new rdflib.Namespace('http://purl.org/linked-data/cube#');
+
+const climateBaseUri = 'http://data.lblod.info/vocabularies/subsidie/climate/';
+const climateTableBaseUri = 'http://data.lblod.info/climate-tables';
+
+const tableEntryBaseUri = 'http://data.lblod.info/id/climate-table/row-entry';
+const ClimateEntryType = new rdflib.NamedNode(`${climateBaseUri}ClimateEntry`);
+const climateEntryPredicate = new rdflib.NamedNode(
+  `${climateBaseUri}climateEntry`
+);
+const climateEntryCustomAction = new rdflib.NamedNode(
+  `${climateBaseUri}customAction`
+);
+const actionDescriptionPredicate = new rdflib.NamedNode(
+  `${climateBaseUri}actionDescription`
+);
+const amountPerActionPredicate = new rdflib.NamedNode(
+  `${climateBaseUri}amountPerAction`
+);
+const restitutionPredicate = new rdflib.NamedNode(
+  `${climateBaseUri}restitution`
+);
+const hasInvalidRowPredicate = new rdflib.NamedNode(
+  `${climateTableBaseUri}/hasInvalidClimateTableEntry`
+);
+const customDescriptionPredicate = new rdflib.NamedNode(
```

Comment:
This is the predicate that is used to store the custom description text, but there might be a better way of doing this? a preflabel for the businessRule maybe?